WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/OrderPaymentSection.swift

```diff
@@ -99,21 +114,14 @@ struct OrderPaymentSection: View {
         }
     }
 
-    @ViewBuilder private var couponRow: some View {
-        if viewModel.shouldShowCoupon {
-            TitleAndValueRow(title: viewModel.couponSummary ?? Localization.coupon, value: .content(viewModel.discountTotal), selectionStyle: .highlight) {
-                shouldShowCouponLineDetails = true
-            }
-            .disabled(viewModel.shouldDisableAddingCoupons)
-        } else {
-            Button(Localization.addCoupon) {
-                shouldShowCouponLineDetails = true
-            }
-            .buttonStyle(PlusButtonStyle())
-            .padding()
-            .accessibilityIdentifier("add-coupon-button")
-            .disabled(viewModel.shouldDisableAddingCoupons)
+    @ViewBuilder private var addCouponRow: some View {
+        Button(Localization.addCoupon) {
+            shouldShowAddCouponLineDetails = true
         }
+        .buttonStyle(PlusButtonStyle())
+        .padding()
+        .accessibilityIdentifier("add-coupon-button")
```

Comment:
We could add these to their own enum:

```suggestion
        .accessibilityIdentifier(Accessibility.addCouponButton)
```

```        
enum Accessibility {
    static let addCouponButton = "add-coupon-button"
}
```

Reply:
Thanks for the comment here. That and the rest of the accessibility identifiers were there before that way, and it's out of the scope of this PR. If you take a look at the app, we also use that approach (plain constantless strings).